src/box/lua/config/instance_config.lua

```diff
@@ -736,10 +691,9 @@ return schema.new('instance_config', schema.record({
         -- The iproto.advertise.{peer,sharding} options have the
         -- following syntax variants:
         --
-        -- 1. user@
-        -- 2. user:pass@
-        -- 3. user@host:port
-        -- 4. host:port
+        -- {login = ..., password = ..., uri = ..., params = ...}
+        --
+        -- where uri is host:port.
```

Comment:
Nit: I would say:

> the following syntax variants:
>
> 1. {login = ...}
> 2. {login = ..., password = ...}
> 3. {uri = ...[, params = ...]}
> 4. {uri = ..., login = ...[, params = ...]}
> 5. {uri = ..., login = ..., password = ...[, params = ...]}

It shows what is allowed and what is not.

(BTW, the item numbers are used in this comment below.)

Or just

> the following syntax:
>
> {login = ..., password = ..., uri = ..., params = ...}

Reply:
True, fixed.